index.html

```diff
@@ -1492,13 +1490,44 @@ <h2 id="state-transitions" class="informative">
         </dd>
       </dl>
     </section>
+    <section data-dfn-for="PaymentComplete" data-link-for="PaymentComplete">
+      <h2><dfn>PaymentComplete</dfn> enum</h2>
```

Comment:
Maybe add the missing "enum" to the PaymentShippingType header while you're here